lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -336,6 +335,7 @@ def add_block(xblock):
             blocks[unicode(xblock.location)] = {
                 u'id': unicode(xblock.location),
                 u'type': xblock.category,
+                u'parent': None,
```

Comment:
What's the effect of this change? If there is a new item in the block, I would expect there to be a test with `None` and not `None` ?